packages/mds-compliance-service/client/index.ts

```diff
@@ -31,5 +31,10 @@ export const ComplianceServiceClient: ServiceClient<ComplianceService> = {
   getComplianceSnapshotsByIDs: (...args) => RpcRequest(ComplianceServiceRpcClient.getComplianceSnapshotsByIDs, args),
   createComplianceSnapshot: (...args) => RpcRequest(ComplianceServiceRpcClient.createComplianceSnapshot, args),
   createComplianceSnapshots: (...args) => RpcRequest(ComplianceServiceRpcClient.createComplianceSnapshots, args),
-  getComplianceViolationPeriods: (...args) => RpcRequest(ComplianceServiceRpcClient.getComplianceViolationPeriods, args)
+  getComplianceViolationPeriods: (...args) =>
+    RpcRequest(ComplianceServiceRpcClient.getComplianceViolationPeriods, args),
+  getComplianceViolation: (...args) => RpcRequest(ComplianceServiceRpcClient.getComplianceViolation, args),
```

Comment:
Can we get a bulk writes endpoint for this entity type?

Reply:
Look good?